apiary.apib

```diff
@@ -825,6 +825,22 @@ Get a quotation
                 + type: `deal` (string)
                 + id: `53474a7a-f9b2-4dd4-88a8-40ce773c7a64` (string)
 
+## Tags [/tags]
+
+### tags.list [GET /tags.list]
+
+Get a list of tags.
+
++ Request (application/json;charset=utf-8)
+
++ Response 200 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `ee6fe9fa-bf92-0d25-9d10-f4e34ba1f7d1` (string)
```

Comment:
Q : I also though we were going to address tags as values, not as entities... or is something blocking that? maybe take this discussion offline?

Reply:
@stivni I had no idea how tags would be tackled but got the task to make the `tags.list` endpoint, so I made a proposal that seemed most logical to me (because they are handled as entities in the database and can be coupled to multiple items).

I don't mind changing this though. It's just a small change.